src/app/routes/assets/verification-method/verification-method.component.ts

```diff
@@ -55,6 +57,15 @@ export class VerificationMethodComponent implements OnInit {
     if (this.isFormDisabled) {
       return;
     }
+
+    const publicKeyExists = this.dataSource.map(key => key.publicKeyHex === this.publicKey.value);
+    if (publicKeyExists) {
+      
+      this.toastr.error('Public key entered already exists, please choose another');
```

Comment:
This `toastr` or i should say `SwitchboardToastrService` is adding notification in header. What i mean is in screenshot. I'm not sure if this is required to have such notifications here. Maybe we should ask Olesia?
![image](https://user-images.githubusercontent.com/83639710/126221106-b1635521-1059-47ee-b847-4ffb905bdf7d.png)
@VOSpdUkraine what do you think?